HiP-DataStore/Core/HistoryUtil.cs

```diff
@@ -51,11 +51,11 @@ public static class HistoryUtil
             }
             catch (SwaggerException e)
             {
-                logger.LogWarning(e,"The request, for getting an access token or the users' details, has been failed. The summary of changes will not show the users' names");
+                logger.LogWarning(e,"The request, for getting the users' details from UserStore, has been failed. The summary of changes will not show the users' names");
```

Comment:
The last small correction 😉: It should be "The request has failed" not "has been failed"

Reply:
It is either: "the request was failed" (past simple: when the period of time is finished or when the time is mentioned) or "the request has been failed" (Present Perfect Simple: To refer to past events that have just happened or been announced or to talk about a past action with a result in the present). So I do not think that I used it wrong!

Reply:
My english is not perfect, but I think you are using the passive formulation here which wrong here.